### PR TITLE
New package: Microsoft.err version 6.4.5

### LINKED ISSUE #113520: [Package Request]: Error Lookup Tool (err.exe)
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### What's the type of your software?

exe

### Please provide the following information

Download Page Url: https://www.microsoft.com/en-us/download/details.aspx?id=100432
Install Options: portable EXE
Publisher: Microsoft Corp.
Publisher Page: 
Package Page:
Package Name:
Description: The Error Lookup Tool, also known as "ERR", is a command-line tool to decode many Microsoft error codes. It does not require elevation and is very easy to use.
Package Version: 6.4.5
Installer URL:
License:
License Page:
Copyright:

### PR DESCRIPTION
**Pull request by egefeyzioglu**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

Resolves #113520

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/124515)

### CODE REVIEW

**Comment by egefeyzioglu** [comment]
@microsoft-github-policy-service agree

**Comment by egefeyzioglu** [comment]
Resolves #113520

File: manifests/m/Microsoft/err/6.4.5/Microsoft.err.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.err
+PackageVersion: 6.4.5
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PackageName: Error Lookup Tool
+License: All Rights Reserved
+Copyright: © Microsoft Corporation. All rights reserved.
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/err/6.4.5/Microsoft.err.locale.en-US.yaml:10]:
```suggestion
License: Proprietary
Copyright: Copyright (c) Microsoft Corporation.
```
A copyright isn't a license

File: manifests/m/Microsoft/err/6.4.5/Microsoft.err.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.err
+PackageVersion: 6.4.5
+PackageLocale: en-US
+Publisher: Microsoft Corporation
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/err/6.4.5/Microsoft.err.locale.en-US.yaml:7]:
```suggestion
Publisher: Microsoft Corporation
PublisherUrl: https://microsoft.com/
PublisherSupportUrl: https://support.microsoft.com/
PrivacyUrl: https://privacy.microsoft.com/privacystatement/
```
Additional metadata to make the manifest more complete

File: manifests/m/Microsoft/err/6.4.5/Microsoft.err.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.err
+PackageVersion: 6.4.5
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PackageName: Error Lookup Tool
+License: All Rights Reserved
+Copyright: © Microsoft Corporation. All rights reserved.
+ShortDescription: The Error Lookup Tool, also known as "ERR", is a command-line tool to decode many Microsoft error codes. It does not require elevation and is very easy to use.
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/err/6.4.5/Microsoft.err.locale.en-US.yaml:11]:
```suggestion
ShortDescription: The Error Lookup Tool, also known as "ERR", is a command-line tool to decode many Microsoft error codes. It does not require elevation and is very easy to use.
Moniker: err
Documentations:
- DocumentLabel: Microsoft Learn Documentation
  DocumentUrl: https://learn.microsoft.com/en-us/windows/win32/debug/system-error-code-lookup-tool
```
Additional metadata to make the manifest more complete

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**